docs/config/directives.md

```diff
@@ -277,7 +277,7 @@ Some common sequence configurations are:
 - _showSequenceNumbers_: boolean
 - _wrap_: boolean
 
-For complete list of sequence diagram configurations, see _defaultConfig.js_ in the source code.
+For complete list of sequence diagram configurations, see _defaultConfig.ts_ in the source code.
```

Comment:
Shall the link be present here as well ?
```suggestion
For complete list of sequence diagram configurations, see [defaultConfig.ts](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/defaultConfig.ts) in the source code.
```

Reply:
@mmorel-35 After running `pnpm run docs:build` this file was modified as well, so I included it in this PR. I took [PR#3707](https://github.com/mermaid-js/mermaid/pull/3707) as an example.

Reply:
@raptor0929, @mmorel-35 asked if we should link that specific defaultConfig.ts too.
Adding modified file was the right thing :).